sys/xtimer/xtimer.c

```diff
@@ -159,22 +159,44 @@ void xtimer_now_timex(timex_t *out)
     out->microseconds = now - (out->seconds * SEC_IN_USEC);
 }
 
-int xtimer_msg_receive_timeout64(msg_t *m, uint64_t timeout) {
-    msg_t tmsg;
-    tmsg.type = MSG_XTIMER;
-    tmsg.content.ptr = (char *) &tmsg;
+/* prepares the message to trigger the timeout
```

Comment:
maybe a dot at the end of the line? or a "-" in front of both? I read this two times as one sentence...